src/GitHub.TeamFoundation.14/Services/LocalRepositoryModelFactory.cs

```diff
@@ -7,7 +7,7 @@ class LocalRepositoryModelFactory : ILocalRepositoryModelFactory
     {
         public ILocalRepositoryModel Create(string localPath)
         {
-            return new LocalRepositoryModel(localPath, GitService.GitServiceHelper);
+            return GitService.GitServiceHelper.CreateLocalRepositoryModel(localPath);
```

Comment:
Future: this is now a part of `IGitService`, we can remove this factory.

Reply:
Done. 👍